src/util/debug.c

```diff
@@ -350,7 +352,16 @@ void ldb_debug_messages(void *context, enum ldb_debug_level level,
     }
 
     if (DEBUG_IS_SET(loglevel)) {
-        sss_vdebug_fn(__FILE__, __LINE__, "ldb", loglevel, APPEND_LINE_FEED,
+        if (level == LDB_DEBUG_TRACE) {
+            /* This is most probably `tevent_debug()` which has own '\n'.
+               As this log level should be virtually never enabled
+               performance considerations (strlen) are not that important.
+             */
+            if (fmt[strlen(fmt)-1] == '\n') {
+                flags = 0;
```

Comment:
Shouldn't we use some NO_FLAGS define (similar to APPEND_LINE_FEED) instead setting flags to 0?

Reply:
I think using 0 as a "no flags" is quite a common practice.

Moreover, there are a couple of places in the SSSD code where 0 was already used.

So I think it is better to open another PR if you think it makes sense to replace 0 with a named constant.

Reply:
OK, agree